pkg/cmd/showConfig.go

```diff
@@ -0,0 +1,59 @@
+package cmd
+
+import (
+	"fmt"
+
+	"github.com/spf13/cobra"
+	"k8s.io/cli-runtime/pkg/genericclioptions"
+	cmdutil "k8s.io/kubectl/pkg/cmd/util"
+	"sigs.k8s.io/yaml"
+
+	"github.com/openshift/microshift/pkg/config"
+)
+
+type showConfigOptions struct {
+	Mode string
+	genericclioptions.IOStreams
+}
+
+func NewShowConfigCommand(ioStreams genericclioptions.IOStreams) *cobra.Command {
+	opts := showConfigOptions{
+		Mode: "default",
+	}
+
+	cfg := config.NewMicroshiftConfig()
+
+	cmd := &cobra.Command{
+		Use:   "show-config",
+		Short: "Print MicroShift's configuration",
+		Run: func(cmd *cobra.Command, args []string) {
+
+			switch opts.Mode {
+			case "default":
+				// Override a few settings to always use the global locations
+				cfg.DataDir = config.DefaultGlobalDataDir
+				cfg.Manifests = cfg.Manifests[:2]
```

Comment:
```suggestion
				cfg.Manifests = cfg.Manifests[:len(cfg.Manifests)-1]
```
Just in case we add other paths in the future, although I don't see that likely..